core/src/surplus.ts

```diff
@@ -86,3 +87,37 @@ export const fetchActiveSurplusAuctions = async function (network: string): Prom
     }
     return surplusAuctions;
 };
+
+export const restartSurplusAuction = async function (
+    network: string,
+    auctionIndex: number,
+    notifier?: Notifier
+): Promise<void> {
+    await executeTransaction(network, 'MCD_FLAP', 'tick', [auctionIndex], {}, notifier);
```

Comment:
Then you can skip it from here:
```suggestion
    await executeTransaction(network, 'MCD_FLAP', 'tick', [auctionIndex], undefined, notifier);
```

Althrough, if we're refactoring this function, maybe it's time to provide all non-required arguments via `options` object? Then you can:
```suggestion
    await executeTransaction(network, 'MCD_FLAP', 'tick', [auctionIndex], { notifier });
```